### ISSUE TITLE
`legacy_connection_handling` setter was deprecated in 7.0 and removed in 7.1

### ISSUE THREAD

**Issue by jpmedforth** [bsm/grape-app#39]
```
An error occurred while loading ./spec/model/store_spec.rb.
Failure/Error: require File.expand_path('../config/environment', __dir__)


ArgumentError:
  The `legacy_connection_handling` setter was deprecated in 7.0 and removed in 7.1, but is still defined in your configuration. Please remove this call as it no longer has any effect."
# /home/john/.rvm/gems/ruby-3.2.2/gems/activerecord-7.1.3/lib/active_record.rb:246:in `legacy_connection_handling='
# /home/john/.rvm/gems/ruby-3.2.2/gems/grape-app-0.11.1/lib/grape/app/initializers/pre.rb:47:in `<top (required)>'
# /home/john/.rvm/gems/ruby-3.2.2/gems/zeitwerk-2.6.12/lib/zeitwerk/kernel.rb:38:in `require'
# /home/john/.rvm/gems/ruby-3.2.2/gems/grape-app-0.11.1/lib/grape/app.rb:31:in `init!'
# ./config/environment.rb:6:in `<top (required)>'
```

**Comment by dim** [comment]
@mxmCherry or @andyborn could one of you pls take a look?

**Comment by jpmedforth** [comment]
@dim this was me. Can you give me permission to push my fix.

**Comment by dim** [comment]
@jpmedforth you should have it, let me try again

**Comment by dim** [comment]
@jpmedforth can you pls take a look?

**Comment by jpmedforth** [comment]
This was corrected with the https://github.com/bsm/grape-app/pull/40 updates.
